### ISSUE TITLE
Add time tracking/logging for db (version 2.1) queries execution time

### ISSUE THREAD

**Issue by ikusteu** [librocco/librocco#517]
The quick implementation is as simple as adding two lines of code:
```ts
const executeQuery = (table: string) => {
	debug.log(ctx, "reactive:notify:table")(table);
	const timerId = `[${q.compile().sql}::${uniqueTimestamp()}]`;
	console.time(timerId); // 1: start timer
	q.execute()
		.then((x) => x || [])
		.then((x) => (console.timeEnd(timerId), x)) //2: log time
		.then(s.next.bind(s));
};
```
But there should be a more sustainable dev friendly solution: be able to turn on/off ideas:
- use context
- use global log level env variables (seems to make more sense, but slightly challenging since db is built before being used in web-client -- check with vite build config)

**Comment by ikusteu** [comment]
Fixed in #520 and subsequently (merged to main) in #512

**Comment by ikusteu** [comment]
Fixed in #520 and subsequently (merged to main) in #512
